### PR TITLE
Engine - Research and enhance suricata integration coverage for ecs compliance

### PR DESCRIPTION
**Pull request by NahuFigueroa97**
|Related issue|
|---|
|#21114|

## Description

The objective of this PR is to conduct a thorough analysis of the current Suricata integration decoders within Wazuh to ensure comprehensive field extraction in alignment with the Elastic Common Schema (ECS). Additionally, we aim to identify and document any significant fields that are not currently represented within ECS but are deemed valuable for security analysis and incident response.

### CODE REVIEW

File: src/engine/ruleset/integrations/suricata/decoders/suricata.yml [hunk 1]
```diff
@@ -14,56 +14,229 @@ metadata:
...
+    map:
+      - url.domain: $destination.domain
+
+  # DNS
+  - check: $network.protocol == dns
+    map:
+      - dns.id: to_string($suricata.dns.id)
+      - dns.response_code: $suricata.dns.rcode
+      - dns.type: $suricata.dns.type
+      - wazuh.dns.question.original: $suricata.dns.rrname
```
Comment (JcabreraC) [inline src/engine/ruleset/integrations/suricata/decoders/suricata.yml:139]: Change field `wazuh.dns.question.original` to `suricata.dns.question`
```suggestion
      - suricata.dns.question.original: $suricata.dns.rrname
```

File: src/engine/ruleset/schemas/wazuh-logpar-types.json [hunk 2]
```diff
@@ -1119,6 +1106,7 @@
         "wazuh.origin": "keyword",
         "wazuh.registered_ip": "keyword",
         "wazuh.source": "keyword",
+        "wazuh.dns.question.original": "text",
```
Comment (JcabreraC) [inline src/engine/ruleset/schemas/wazuh-logpar-types.json:1109]: Change field `wazuh.dns.question.original` to `suricata.dns.question`
```suggestion
        "suricata.dns.question.original": "text",
```

**Review by JcabreraC — CHANGES_REQUESTED** [review]

File: src/engine/ruleset/schemas/wazuh-logpar-types.json (hunk 2 above)
Comment (NahuFigueroa97) [reply src/engine/ruleset/schemas/wazuh-logpar-types.json]: this information already exist in dns.question.name

**Review by JcabreraC — APPROVED** [review]
LGTM !

**Review by JavierBejMen — APPROVED** [review]
LGTM

**Pull request merged**